### DISCUSSION TITLE
Ajouter la convention collective syntec

### DISCUSSION THREAD

**Discussion by johangirod** [betagouv/mon-entreprise#2655] [Convention collective]
Convention collective relativement connue.
https://code.travail.gouv.fr/convention-collective/1486-bureaux-detudes-techniques-cabinets-dingenieurs-conseils-et-societes-de

**Comment by mquandalle** [comment]
(À noter qu'il y a déjà une référence à la convention Syntec dans le calcul de l'activité partielle, il faudra donc regrouper cette référence.)
